uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/config/drivetrain_controllers.yaml

```diff
@@ -0,0 +1,61 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 100 # Hz
+
+    joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+
+    differential_drivetrain_controller:
+      type: diff_drive_controller/DiffDriveController
+
+differential_drivetrain_controller:
+  ros__parameters:
+    left_wheel_names: [ "left_front_wheel_joint", "left_middle_wheel_joint", "left_back_wheel_joint" ]
+    right_wheel_names: [ "right_front_wheel_joint", "right_middle_wheel_joint", "right_back_wheel_joint" ]
+
+    # Should match values defined in uwrt_mars_rover_drivetrain_description's generated urdf
+    wheel_separation: 1.2 # wheelbase_length = chassis_width + wheel_width
+    wheels_per_side: 3
+    wheel_radius: 0.15
+
+    wheel_separation_multiplier: 1.0
+    left_wheel_radius_multiplier: 1.0
+    right_wheel_radius_multiplier: 1.0
+
+    odom_frame_id: odom
+    base_frame_id: base_link
+    pose_covariance_diagonal: [ 0.001, 0.001, 0.001, 0.001, 0.001, 0.01 ]
+    twist_covariance_diagonal: [ 0.001, 0.001, 0.001, 0.001, 0.001, 0.01 ]
+
+#    publish_rate: 50.0  # Available once https://github.com/ros-controls/ros2_controllers/pull/278 merges
```

Comment:
this has been merged already. just waiting on binary release

Reply:
Version 1.3.0 has released and has been synced upstream:
https://github.com/ros-controls/ros2_controllers/blob/master/diff_drive_controller/CHANGELOG.rst
https://index.ros.org/p/diff_drive_controller/github-ros-controls-ros2_controllers/#galactic